keyboards/pdxkbc/readme.md

```diff
@@ -0,0 +1,17 @@
+# pdxkbc
+
+![pdxkbc](https://i.imgur.com/GgNvZcW.jpg)
+
+A macropad made for the Portland Keyboard Club and DEF CON
+
+Keyboard Maintainer: [Franklin Harding](https://github.com/yourusername)  
+
```

Comment:
By "two spaces" I meant the kind you get by hitting the spacebar twice. :smile: 

```suggestion
```

Reply:
Right, but github renders this correctly now, doesn't it?

Reply:
I prefer this style of spacing since the spacing is much more explicit looking at the raw markdown.

Reply:
We can leave it like this.

Reply:
Thanks!